resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -27,15 +27,19 @@ private[spark] class PythonStep(
     filesDownloadPath: String) extends KubernetesSubmissionStep {
 
   override def prepareSubmission(driverSpec: KubernetesDriverSpec): KubernetesDriverSpec = {
+    val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
```

Comment:
I am testing that rn in a seperate PR, but yes, it should. passing in just `null` I believed cause withValue() to raise an error. But I think that CMD PythonRunner PY_FILE "null" DRIVER_ARGS parses correctly.